src/runner/driver-test-runner.js

```diff
@@ -1,19 +1,39 @@
 import { startJob } from '../shared/job.js';
 
-import { ATDriver, ATKey, webDriverCodePoints } from './at-driver.js';
+import { ATKey, webDriverCodePoints } from './at-driver.js';
 import { RunnerMessage } from './messages.js';
 
 /**
- * @module agent
+ * @module runner
  */
 
+export const NVDASettingResponses = {
+  browseMode: ['Browse mode'],
+  focusMode: ['Focus mode'],
+};
+
+export const VOSettingResponses = {
+  quickNavOn: ['Quick nav on', 'All quick nav on'],
```

Comment:
Maybe add a note about macOS/VoiceOver versions here or in the commit message. (As it stands, folks will have to dig through to the originating bug report to understand the motivation.)